src/btree/bt_handle.c

```diff
@@ -1104,10 +1105,11 @@ __btree_page_sizes(WT_SESSION_IMPL *session)
     WT_RET(__wt_config_gets(session, cfg, "memory_page_max", &cval));
     btree->maxmempage = (uint64_t)cval.val;
 
+    dirty_trigger = __wt_atomic_load_double_relaxed(&conn->evict->eviction_dirty_trigger);
```

Comment:
I hope all compilers should optimise it, but wouldn't it be clearer to put this under the if where we use this value?